src/referencePackages/src/system.text.json/8.0.0/lib/netstandard2.0/System.Text.Json.cs

```diff
@@ -1747,15 +1747,15 @@ public partial interface IJsonTypeInfoResolver
 
     public sealed partial class JsonCollectionInfoValues<TCollection>
     {
-        public JsonTypeInfo ElementInfo { get { throw null; } init { } }
+        public JsonTypeInfo ElementInfo { get { throw null; } set { } }
```

Comment:
Is this a GenAPI bug? cc @ericstj @andriipatsula

Reply:
That should have been fixed long ago : https://github.com/dotnet/roslyn/pull/68960

I checked with the latest GenAPI commandline tool and was able to get it to emit the correct `init` accessor for both the netstandard2.0 build and the net8.0 build of STJ.